### PR TITLE
Feature/ob 19 cosmetics

### PR DESCRIPTION
**Pull request by Alinochka07**
Component is created. It redirects to a new page called "cosmetics".

### CODE REVIEW

**Comment by Alinochka07** [comment]
Also I made some changes in styling in other files too, to avoid "double" adding margins and paddings to other components...

File: src/components/PromoOrangeHeader/PromoHeader.tsx [hunk 1]
```diff
@@ -1,31 +1,27 @@
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
```
Comment (Alex-2kZharkov) [inline src/components/PromoOrangeHeader/PromoHeader.tsx]: You can omit this import

File: src/components/PromoOrangeHeader/PromoHeader.tsx [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 
-export const PromoHeader = () => {
+type PromoOrangeHeader = {
```
Comment (Alex-2kZharkov) [inline src/components/PromoOrangeHeader/PromoHeader.tsx]: Please check with naming convention for component props

Comment (Alex-2kZharkov) [reply src/components/PromoOrangeHeader/PromoHeader.tsx]: Actually, you can omit creating your own type. Use built-in  `PropsWithChildren` from react

File: src/components/PromoOrangeHeader/PromoHeader.tsx [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
+  children: React.ReactNode;
+};
+
+export const PromoHeader = (props: PromoOrangeHeader): React.JSX.Element => {
```
Comment (Alex-2kZharkov) [inline src/components/PromoOrangeHeader/PromoHeader.tsx]: Use `ReactElement` as a return type

Comment (Alex-2kZharkov) [reply src/components/PromoOrangeHeader/PromoHeader.tsx]: + don't forget to destructure the `children`

File: package.json [hunk 4]
```diff
@@ -20,11 +20,12 @@
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
     "postcss": "8.4.24",
+    "prop-types": "^15.8.1",
```
Comment (Alex-2kZharkov) [inline package.json]: I don't think we need this library; we use TypeScript for such things.

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: src/components/PromoOrangeHeader/PromoHeader.tsx (hunk 2 above)
Comment (Alinochka07) [reply src/components/PromoOrangeHeader/PromoHeader.tsx]: okay, I've destructured this part.

File: src/components/PromoOrangeHeader/PromoHeader.tsx (hunk 3 above)
Comment (Alinochka07) [reply src/components/PromoOrangeHeader/PromoHeader.tsx]: done

File: package.json (hunk 4 above)
Comment (Alinochka07) [reply package.json]: removed

**Comment by Alinochka07** [comment]
changes are applied.

**Comment by Alinochka07** [comment]
fixed all conflicts

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**